core/tests/protractor_desktop/voiceoverUploadFeatures.js

```diff
@@ -85,6 +89,49 @@ describe('Voiceover upload features', function() {
       '../data/cafe-over-five-minutes.mp3');
     explorationEditorTranslationTab.expectSaveUploadedAudioButtonToBeDisabled();
     explorationEditorTranslationTab.closeUploadAudioModal();
+    explorationEditorTranslationTab.deleteAudioRecord();
+  });
+
+  it('should upload recorded audio', function() {
+    explorationEditorTranslationTab.addAudioRecord();
+    explorationEditorTranslationTab.stopAudioRecord();
+    explorationEditorTranslationTab.confirmAudioRecord();
+    explorationEditorTranslationTab.playAudioRecord();
+    browser.refresh();
+    explorationEditorTranslationTab.playAudioRecord();
+  });
+
+  it('should play recorded file after logging out', function() {
+    users.logout();
+    users.login(TEST_EMAIL);
+    creatorDashboardPage.get();
+    creatorDashboardPage.editExploration('Untitled');
+    explorationEditorMainTab.exitTutorial();
+
+    explorationEditorPage.navigateToTranslationTab();
+    explorationEditorTranslationTab.playAudioRecord();
```

Comment:
This assumes the previous test ran before this? It might not always be the case I think (due to parallelization). Would be better to keep it separate from the above test.

Reply:
Resolved by 70592c2e6